app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -147,10 +149,35 @@ const classNameStory = (storyType, estimate) => classname (
   }
 );
 
-const StoryItem = ({ title, storyType, estimate, labels, state, ownedByInitials, ownedByName }) => (
+const StoryItem = ({
+  title,
+  storyType,
+  estimate,
+  labels,
+  state,
+  description,
+  ownedByInitials,
+  ownedByName,
+  requestedByName,
+  createdAt,
+  notes
+}) => (
   <div className={classNameStory(storyType, estimate)}>
-    <StoryIcon storyType={storyType} />
-    <StoryEstimate estimate={estimate} />
+    <StoryPopover
+      description={description}
+      notes={notes}
+      createdAt={createdAt}
+      title={title}
+      storyType={storyType}
+      requestedByName={requestedByName}
+    >
+      <div className={'Story__icons-block'}>
+        <StoryIcon storyType={storyType} />
+        <StoryEstimate estimate={estimate} />
+        <StoryDescriptionIcon description={description}/>
```

Comment:
What if you use a conditional render instead of this flag argument?
```suggestion
        description && <StoryDescriptionIcon />
```

Reply:
I don't agree with that, it breaks the [tell don't ask](https://martinfowler.com/bliki/TellDontAsk.html) principle. Otherwise we would have to do this external check everytime we render this component.

Reply:
I still think it's better to use conditional rendering even though we are breaking the principle because in this case, StoryDescriptionIcon will only render if the StoryItem has a description.
I don't see flag arg as a good practice especially because StoryDescriptionIcon should render its icon without doing any checks.  :thinking: 
This conditional rendering will depend only on whether StoryItem is optional or not.